### PR TITLE
Catch error raised by already-deleted partition

### PR DESCRIPTION
**Pull request by mdurrant-b3**
Fixes issue caused when the last element in an LB is deleted and the associated partition is deleted with it.

### CODE REVIEW

**Comment by mdurrant-b3** [comment]
check

**Comment by mdurrant-b3** [comment]
pulls-ac-double-lbaasv1-dsvm-api-acos-272 failed on initial CI run.

**Comment by mdurrant-b3** [comment]
An alternative implementation for this would be:
modify partition_cleanup_check to return True/False if it deleted the partition and pass this value to the `super`'s `__exit__` function to make the "write" (sorry couldn't resist the pun) decision.

File: acos_client/v21/action.py [hunk 1]
```diff
@@ -46,5 +46,9 @@ def activate_and_write(self, partition, **kwargs):
             # Catch 'might fail error'
             if e.msg.startswith("write memory"):
                 pass
+                # Get partition 'cdc164edcb4f4': Partition does not exist.
+            # The partition no longer exists
+            if "Get partition '{0}': Partition does not exist.".format(partition) in e.msg:
```
Comment (dougwig) [inline acos_client/v21/action.py]: Yuck, -1

**Comment by mdurrant-b3** [comment]
check

**Pull request merged**